src/datachain/sql/sqlite/base.py

```diff
@@ -217,23 +217,6 @@ def register_user_defined_sql_functions() -> None:
     # Register optional functions if we have the necessary dependencies
     # and otherwise register functions that will raise an exception with
     # installation instructions
-    try:
```

Comment:
from memory: we need this as a fallback because usearch is not available on Windows - https://github.com/unum-cloud/usearch/issues/427

Reply:
🤷🏻 - https://github.com/iterative/dvcx/pull/1679/files#r1664972655

Reply:
description on that PR should shed some light on the approach